html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+      <div>
+        <form action="index.html">
+          <p>Email: <br>
```

Comment:
Instead of using a `<p>` tag for the input text label it should be a `<label>` tag:

```HTML
<label>Email:</label>
```

Also instead of using a `<br>` tag it would be better to use a `<div>`:

```HTML
<div>
  <label for="email-address">Email</label>
  <div>
    <input name="email" type="email" id="email-address">
  </div>
</div>
```